scripts/utils.sh

```diff
@@ -134,11 +127,23 @@ function close_external_ports() {
     ports=$1
     for p in $ports; do
         sudo firewall-cmd --zone=public --remove-port=$p/tcp
+        if [ ! -z ${PLATFORM} ]; then
```

Comment:
nit: why not ?
```
if [ ! -z "${PLATFORM}" ] && ([[ "${PLATFORM}" == "none" || "${PLATFORM}" == "external" ]]); then
    sudo firewall-cmd --policy=libvirt-to-host --remove-port=$p/tcp
fi
```

Reply:
easier to read, it’s exactly the same

Reply:
eitherway , is not going to work the file is configured with `nounset` so we have to set it